python/worker.py

```diff
@@ -0,0 +1,38 @@
+"""Main entrypoint."""
+
+
+import argparse
+import os
+import sys
+import traceback
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Python Azure Functions Worker')
+    parser.add_argument('--host')
+    parser.add_argument('--port', type=int)
+    parser.add_argument('--workerId', dest='worker_id')
+    parser.add_argument('--requestId', dest='request_id')
```

Comment:
Do we see a friendly error message if any of the arguments are missing?

Reply:
Yes, the argparse module takes care of that.